utils/appliance.py

```diff
@@ -114,6 +117,14 @@ def db(self):
     def version(self):
         return self.ipapp.version
 
+    @lazycache
+    def backup_database(self):
+        return self.ipapp.backup_database
+
+    @lazycache
+    def restore_database(self):
+        return self.ipapp.restore_database
```

Comment:
You do not need to lazycache these two methods.

```python
def backup_database(self, *args, **kwargs):
    return self.ipapp.backup_database(*args, **kwargs)
```

@seandst I actually think some DRY-ing might be good here, something like `SimpleDelegator` in Ruby so we don't have to write all these simple accessors.